pr-data.csv

```diff
@@ -1932,14 +1932,15 @@ https://github.com/apache/hbase,881c92b892844be567d5f26b161a820ebf319f84,hbase-z
 https://github.com/apache/hbase,881c92b892844be567d5f26b161a820ebf319f84,hbase-zookeeper,org.apache.hadoop.hbase.zookeeper.TestZKUtil.testGetChildDataAndWatchForNewChildrenShouldNotThrowNPE,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/37
 https://github.com/apache/hbase,881c92b892844be567d5f26b161a820ebf319f84,hbase-zookeeper,org.apache.hadoop.hbase.zookeeper.TestZKUtil.testSetDataWithVersion,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/37
 https://github.com/apache/hbase,881c92b892844be567d5f26b161a820ebf319f84,hbase-zookeeper,org.apache.hadoop.hbase.zookeeper.TestZKUtil.testZNodeDeletes,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/37
-https://github.com/apache/helix,917b786e4da9a615c5177f27256e170feae2298f,helix-rest,org.apache.helix.rest.metadatastore.TestZkMetadataStoreDirectory.testGetAllShardingKeys,ID,,,
+https://github.com/apache/helix,917b786e4da9a615c5177f27256e170feae2298f,helix-rest,org.apache.helix.rest.metadatastore.TestZkMetadataStoreDirectory.testGetAllShardingKeys,ID,Opened,https://github.com/apache/helix/pull/2666,
 https://github.com/apache/helix,917b786e4da9a615c5177f27256e170feae2298f,helix-rest,org.apache.helix.rest.server.json.cluster.TestClusterInfo.whenSerializingClusterInfo,ID,Accepted,https://github.com/apache/helix/pull/2297,
 https://github.com/apache/helix,917b786e4da9a615c5177f27256e170feae2298f,helix-rest,org.apache.helix.rest.server.json.cluster.TestClusterTopology.whenSerializingClusterTopology,ID,Accepted,https://github.com/apache/helix/pull/2297,
 https://github.com/apache/helix,917b786e4da9a615c5177f27256e170feae2298f,helix-rest,org.apache.helix.rest.server.json.instance.TestStoppableCheck.whenSerializingStoppableCheck,ID,Accepted,https://github.com/apache/helix/pull/2297,
-https://github.com/apache/helix,917b786e4da9a615c5177f27256e170feae2298f,helix-rest,org.apache.helix.rest.server.TestInstancesAccessor.testGetAllInstances,ID,,,
+https://github.com/apache/helix,917b786e4da9a615c5177f27256e170feae2298f,helix-rest,org.apache.helix.rest.server.TestInstancesAccessor.testGetAllInstances,ID,Opened,https://github.com/apache/helix/pull/2665,
+https://github.com/apache/helix,6241bcbf5f9fab1726c87c20b5e948daa95b7bbf,helix-rest,org.apache.helix.rest.server.TestPerInstanceAccessor.testGetAllInstances,OD,Claimed,,
```

Comment:
Please remove this line with Claimed. The rest is ready to be accepted.

Reply:
done